pyproject.toml

```diff
@@ -26,24 +26,24 @@ classifiers=[
     "Programming Language :: Python :: 3.11",
 ]
 dependencies = [
-"numpy",
-"accelerate>=0.20.3",
-"transformers",
-"torch",
-"sentencepiece",
-"tokenizers>=0.13.3",
-"tqdm",
-"trl",
-"peft>=0.8.0",
-"datasets>=2.15.0",
-"fire",
-"simpleeval",
+"numpy>=1.26.4,<2.0",
+"accelerate>=0.20.3,<1.0",
+"transformers>=4.34.1,<5.0,!=4.38.2",
+"torch>=2.2.0,<3.0",
+"sentencepiece>=0.1.99,<1.0",
```

Comment:
perhaps set more conservative limit say 0.3  (current 0.2). This package updates very slow.